apps/microtvm/gemmini/template_project/src/dwconv2d.c

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
same comment for all of these.

Reply:
Yes, I applied the suggestion for all of them.